client/state/data-layer/wpcom/sites/rewind/index.js

```diff
@@ -1,7 +1,49 @@
 /** @format */
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
 /**
  * Internal dependencies
  */
+import { mergeHandlers } from 'state/action-watchers/utils';
+import { REWIND_STATE_REQUEST, REWIND_STATE_UPDATE } from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequestEx, makeParser } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'state/notices/actions';
+import { transformApi } from './api-transformer';
+import { rewind } from './schema';
+
 import downloads from './downloads';
 
-export default downloads;
+const fetchRewindState = action =>
+	http(
+		{
+			apiVersion: '1.1',
+			method: 'GET',
+			path: `/sites/${ action.siteId }/rewind`,
+			query: {
+				force: 'wpcom',
+			},
+		},
+		action
+	);
+
+const updateRewindState = ( { siteId }, data ) => ( {
+	type: REWIND_STATE_UPDATE,
+	siteId,
+	...data,
+} );
+
+export default mergeHandlers( downloads, {
+	[ REWIND_STATE_REQUEST ]: [
+		dispatchRequestEx( {
+			fetch: fetchRewindState,
+			onSuccess: updateRewindState,
+			onError: () =>
+				errorNotice( translate( 'Failed to fetch Rewind information, please contact support.' ) ),
```

Comment:
Should we have a link to support here?

Reply:
not sure. probably could use some help from @MichaelArestad here - if we can't get Rewind state, what do we say? Currently I think we just say to try again.

Reply:
@dmsnell We should have a link to support (chat?) as well as try again. Otherwise, it might make sense to allow Try again for one more try before we show a contact support link instead?

Let's also be a bit more clear with the message. When does this message show up?

Reply:
If the API call fails this will appear and it could eventually appear on the settings page where credentials are or on the activity log page.

Reply:
> Try again for one more try before we show a contact support link instead?

Of note, Calypso auto-retries here by default up to three times so by this point it's probably not going to succeed on a retry.

Reply:
we could just ignore failures too and rely on the components to note failures. I'm not sure what is right here.